geopandas/geoseries.py

```diff
@@ -53,6 +53,37 @@ def _geoseries_constructor_with_fallback(data=None, index=None, crs=None, **kwar
         return Series(data=data, index=index, **kwargs)
 
 
+def _geoseries_expanddim(data=None, index=None, crs=None, **kwargs):
+    from geopandas import GeoDataFrame
+
+    # DataFrame == pd.Series._constructor_expanddim
```

Comment:
```suggestion
    # pd.Series._constructor_expanddim == pd.DataFrame
```

(small nit, but reads a bit more logical to me)

Reply:
Yep, it took two reads for me to work out what this comment meant, and I wrote, that's a bad sign.